apparmor.d/abstractions/bwrap-app

```diff
@@ -10,7 +10,8 @@
 
   include <abstractions/audio>
   include <abstractions/consoles>
-  include <abstractions/dbus-session>
+  include <abstractions/bus-session>
+  include <abstractions/bus-system>
```

Comment:
You need to the rules:
```
dbus bus=session,
dbus bus=system,
```

Otherwise it will break a lot of things. We may improve this later.